crypto/dilithium/sig_dilithium.h

```diff
@@ -29,6 +29,8 @@ int DILITHIUM3_keypair(uint8_t *public_key,
 int DILITHIUM3_sign(uint8_t *sig, size_t *sig_len,
                     const uint8_t *message,
                     size_t message_len,
+                    const uint8_t *ctx,
```

Comment:
Should the docstrings be updated to describe |ctx|?

Reply:
Added in [bfffd27](https://github.com/aws/aws-lc/pull/1894/commits/bfffd27baff55ce003bb017a8e95b2eb079e8216) :)